gapic-generator/templates/default/service/client/_client.erb

```diff
@@ -56,6 +70,20 @@ class <%= service.client_name %>
   ##
   # Create a new <%= service.client_name %> client object.
   #
+  # ## Examples
+  #
+  # To create a new <%= service.client_name %> client with the default
```

Comment:
I would change `service.client_name` to `service.name`. Currently the output for all gems (below) is `To create a new Client client`.

You can also fix this on line 71 and possibly in other places.

Reply:
Fixed